src/domains/wallet/pages/WalletDetails/WalletDetails.tsx

```diff
@@ -77,9 +77,10 @@ export const WalletDetails = ({ txSkeletonRowsLimit, transactionLimit }: WalletD
 	useLayoutEffect(() => {
 		setShowWalletVote(activeWallet.network().can("Transaction.vote"));
 		setShowWalletRegistrations(
-			activeWallet.network().can("Transaction.secondSignature") ||
-				activeWallet.network().can("Transaction.delegateRegistration") ||
-				activeWallet.network().can("Transaction.entityRegistration"),
+			!activeWallet.isLedger() &&
+				(activeWallet.network().can("Transaction.secondSignature") ||
```

Comment:
See `1.1.9` and https://github.com/ArkEcosystem/platform-sdk/commit/684bca7fe525efb6fef44cd0c672cc403961b548